docs/platform/concepts/service-power-cycle.rst

```diff
@@ -0,0 +1,39 @@
+Service power cycle
+===================
+
+Aiven service power off and power on is more than stopping and starting a service on nodes. For better utilisation of resources on Aiven platform, idle resources will be released and only the necessary data will be kept after power off. The impact on the service is different depending on the service type and plan. 
+
+.. warning:: Data loss may happen during a service power off, so it is important for users to understand the consequences before powering off a service.
+
+Aiven service power off and power on can be done on `Aiven Console <https://console.aiven.io>`_ or through :doc:`Aiven CLI </docs/platform/howto/pause-from-cli>`.
+
+Power off
+-------------
+
+After an Aiven service is powered off:
+
+* All virtual machine(s) of the service will be **removed** from the public cloud.
+* The service information and configuration will be stored on Aiven Platform, while service data will be lost if there's no backup available .
```

Comment:
Again I would state for which services this is a problem

Reply:
And I will put this sentence AFTER the **time-based**  description

Reply:
That section is for ALL services. 

The first thing users need to be aware is the VMs will be gone, then they need to understand what will be kept for the service by reading the following bullets.